source-zendesk-support/VERSION

```diff
@@ -0,0 +1 @@
+v3
```

Comment:
The existing connector is `v1` I believe, and I think it makes sense to do a version bump, so this should probably be `v2` instead of `v3`.

Reply:
You're right that the existing connector is `v1`. When we briefly talked about how we were going to roll out the imported connector, I thought you mentioned bringing the `v1` connector into the `connectors` repo & renaming it to `v2`. Then the imported connector would be `v3`, and we could roll back to `v2` if there were issues with `v3`. Am I misremembering? Or maybe there's a better roll out strategy?

Reply:
We can bring the `v1` connector into the connectors repo as `v1` also, so we'll only need to go to `v2`

Reply:
Got it, I'll change this one to `v2`. Thanks!